kobo/apps/project_ownership/tasks.py

```diff
@@ -264,16 +264,42 @@ def task_restarter():
     stuck_threshold = timezone.now() - timedelta(
         minutes=config.PROJECT_OWNERSHIP_STUCK_THRESHOLD
     )
+    restarted_transfers = []
+    for transfer_status in TransferStatus.objects.filter(
+        date_modified__lte=resume_threshold,
+        status=TransferStatusChoices.PENDING,
+        status_type=TransferStatusTypeChoices.GLOBAL,
+    ):
+        # restart any transfers that were left in pending but not picked up
+        transfer_status.transfer.transfer_project()
+        restarted_transfers.append(transfer_status.transfer)
```

Comment:
For any reason, we have _n_ (where _n_ is a high number) projects to restart in this loop, it could take a while. The other loop could never make it. 

Any possibility to avoid to restart transfers sequentially?

Reply:
Some options:
1. Put a hard limit on this loop the same way we do with MAX_RESTARTED_TASKS
2. Spawn new tasks that call transfer_project rather than doing it directly in the loop
3. Have a separate task altogether for calling transfer project and be very careful we don't accidentally pick up the same tasks twice
4. Do this loop second 

1 and 4 are the easiest, followed by 2 and then 3.